lib/config/definitions.js

```diff
@@ -1441,6 +1442,21 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'poetry',
+    releaseStatus: 'unpublished',
+    description: 'Configuration object for pyproject.toml files',
+    stage: 'package',
+    type: 'toml',
+    default: {
+      enabled: false,
+      commitMessageTopic: 'Poetry package {{depName}}',
+      managerBranchPrefix: 'poetry-',
```

Comment:
We probably don't want a manager prefix because if a repository used different python managers but shared dependencies then we want them in the same branch